src/spec/Fastly/Adapter/Guzzle/GuzzleAdapterSpec.php

```diff
@@ -7,12 +7,12 @@
 
 class GuzzleAdapterSpec extends ObjectBehavior
 {
-    function it_is_initializable()
+    public function it_is_initializable()
```

Comment:
@thommahoney 

About missing visibility, not sure if `phpspec/phpspec` has been updated and provides visibility now, giving this a check.